readme.md

```diff
@@ -2,3 +2,59 @@ Ansible Atlantis Role
 ========================
```

Comment:
```suggestion
# Ansible Atlantis Role
```

We switched to this style of the first heading, because the text is not always the same length and so this is more consistent.